components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -213,17 +213,22 @@ pub enum AnimatedProperty {
 }
 
 impl AnimatedProperty {
-    /// Get the name of this property.
-    pub fn name(&self) -> &'static str {
+    /// Get the id of the property we're animating.
+    pub fn id(&self) -> LonghandId {
         match *self {
             % for prop in data.longhands:
-                % if prop.animatable:
-                    AnimatedProperty::${prop.camel_case}(..) => "${prop.name}",
-                % endif
+            % if prop.animatable:
+            AnimatedProperty::${prop.camel_case}(..) => LonghandId::${prop.camel_case},
+            % endif
```

Comment:
I wonder if `AnimatedProperty` should use the same trick as `AnimationValue` to just read the discriminant value as the `LonghandId` result.